build-tools/packages/build-cli/src/commands/bump.ts

```diff
@@ -54,7 +54,6 @@ export default class BumpCommand extends BaseCommand<typeof BumpCommand> {
 		exactDepType: Flags.string({
 			description:
 				"When using the exact flag, controls the type of dependency that is used between packages within the release group.",
```

Comment:
What does this do when not using the --exact flag? I think this doc needs updating.

Reply:
Oh, I see, it does nothing, but having a default breaks